### DISCUSSION TITLE
Error: accepts 1 arg(s), received 4

### DISCUSSION THREAD

**Discussion by satriazoid** [JanDeDobbeleer/oh-my-posh#5879] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I found this problem when opening powershell for the first time, what should I do?
![Screenshot_256](https://github.com/user-attachments/assets/91bedb3b-163e-4c2d-bd56-d172e8404fe6)

### Theme

dark

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Line |
   1 |  oh-my-posh debug <shell> --plain
     |                   ~
     | The '<' operator is reserved for future use.
```

**Comment by JanDeDobbeleer** [comment]
@satriazoid what line did you add to your profile to initialize oh-my-posh?
